internal/importers/base/importer_test.go

```diff
@@ -382,10 +366,11 @@ func TestBaseImporter_ImportProjects(t *testing.T) {
 				},
 			},
 		},
-		"ignore default source": {
+		"skip default source": {
```

Comment:
Let's add a new test case that validates the logic to preserve an empty (any) constraint when an alternate source is set.

Reply:
I think this is satisfied by the `alternate source` test above.

Reply:
🤦‍♀️ Doh, I completely missed that the first time through! er... carry on!